bin/checkpassword

```diff
@@ -82,13 +83,38 @@ if [ "$?" -eq 0 ]; then
 	export userdb_gid=5000
 	export EXTRA="userdb_uid userdb_gid $EXTRA"
 
-    # add quota
-    quota=$(${QUOTA_COMMAND} "$USER")
+    # add quota settings
+    quota="$(${QUOTA_COMMAND} "$USER")"
     if [ -n "$quota" ]; then
         export userdb_quota_rule="*:storage=${quota}M"
 	    export EXTRA="userdb_quota_rule $EXTRA"
     fi
 
+    # add mail_crypt settings
+    if [ "$AUTHORIZED" != 2 ]; then
```

Comment:
You're correct about Dovecot. But checkpassword sets `$AUTHORIZED=2` earlier if `$AUTHORIZED` was `1`. So at this point in code, `$AUTHORIZED` is set to `2`. See lines 64-70 of `bin/checkpassword`.

Reply:
It would be cleaner to register a var in 64-70 and check for this here. You Wouldn't know if dovecot provided other values of `AUTHORIZED` for other special use cases.

Reply:
Agreed, you're correct. I'll introduce a new internal variable `USERDB_LOOKUP` for that.